cookbooks/muparser_temperature_example/muparser-temperature-example.prm

```diff
@@ -61,12 +61,12 @@ end
 
 # Refine the upper 150 km of the mesh so lithosphere structure is resolved.
 # Function expression asks if the depth is less than the lithosphere depth and
-# then refinement level = 7, else refinement level = 0. 
-# Note, the minimum refinement level for the lower 850 km of the mesh would be 
-# coarser than the initial one level of global refinement. This would then lead
+# then refinement level = 7, else refinement level = 3. 
+# Note, if the minimum refinement level for the lower 850 km of the mesh would be 
+# less than the initial global refinement level, then this would lead
```

Comment:
Did you actually observe this case? It seems like a bug that the minimum refinement function would overwrite the general parameter `Minimum refinement level`. I was not aware this is happening.